src/slurm.jl

```diff
@@ -50,44 +50,46 @@ function launch(manager::SlurmManager, params::Dict, instances_arr::Array,
             mkdir(job_file_loc)
         end
 
-        println("removing old files")
+        # println("removing old files")
         # cleanup old files
-	map(f->rm(joinpath(job_file_loc, f)), filter(t -> occursin(r"job(.*?).out", t), readdir(job_file_loc)))
-        println("removing old Setting up srun commands")
+	    # map(f->rm(joinpath(job_file_loc, f)), filter(t -> occursin(r"job(.*?).out", t), readdir(job_file_loc)))
 
         np = manager.np
         jobname = "julia-$(getpid())"
```

Comment:
I wonder if it would be worth making this user-setable, with this as default. Probably beyond the scope of this PR...

Reply:
I was thinking that, too.  I could expose that functionality from `launch`, but I think it would be a separate PR to expose it to the end-user.  Which would be fine, but I think there needs to be some discussion of what form that option would take.  A boolean could be fine, but to really be complete you'd probably want the option to be an anonymous function, like `remove_old_logs=t -> occursin(r"job(.*?).out", t)` maybe with the convenience function that `remove_old_logs=true` uses the function `remove_old_logs=t -> occursin(r"job(.*?).out", t)` by default.  And `remove_old_logs=false` does nothing.  I'll open an issue for this.

Reply:
Huh. Seems like this wouldn't need to be standardized-- it just flows through the params argument to launch. I'll add it as i described in the previous comment, unless anyone has any objections. (so it'll be a keyword argument to `addprocs_slurm` which will be passed to `launch` via the `params` dict)

Reply:
In testing this out, I changed my mind.  I'm not comfortable adding this feature in this way because it's not obvious what is meant by an "old" log file.  In particular, if you run `addprocs_slurm` a second time in a single session, with this option set to `true`, it will delete the still-active log files from the first session.  Since it's easy enough to delete them manually, I'd rather not introduce the danger and ambiguity.